gallery/src/styles/NxFormLayout/NxFormLayoutExample.tsx

```diff
@@ -78,8 +78,8 @@ export default function NxFormLayoutExample() {
       <fieldset className="nx-fieldset">
         <legend className="nx-legend nx-legend--optional">
           <span className="nx-legend__text">Primary Color</span>
-          <span className="nx-sub-label">Pick a single color</span>
         </legend>
+        <div className="nx-sub-label">Pick a single color</div>
```

Comment:
I was under the impression that this was inside the `legend` on purpose.  It makes sense that way to me.  The recently-merged `NxFieldset` also is implemented with the `.nx-sub-label` within the `.nx-legend`.

Reply:
> I was under the impression that this was inside the `legend` on purpose. It makes sense that way to me. The recently-merged `NxFieldset` also is implemented with the `.nx-sub-label` within the `.nx-legend`.

Mostly I think that `.sub-label` should be consistently either inside or outside of labels and legends, I'm not sure that it matters which and I can switch all of the examples to inside.

Reply:
> Mostly I think that .sub-label should be consistently either inside or outside of labels and legends, I'm not sure that it matters which and I can switch all of the examples to inside.

Originally, the label text, sublabel, and input were all inside of the label.  Then for screenreader support we changed it so the input and sublabel were outside of the label.  Maybe we didn't have to move the sublabel at that time, I'm not sure.  But as it stands that history means we are already supporting two different ways of doing them, I'd rather not add a third by moving the sublabel back inside of labels.  If you want them consistent I'd rather move legend sublabels to the outside.  But if that's what you do it needs to be changed in `NxFieldset` as well.

Reply:
> I was under the impression that this was inside the `legend` on purpose. It makes sense that way to me. The recently-merged `NxFieldset` also is implemented with the `.nx-sub-label` within the `.nx-legend`.

Moved back inside here: https://github.com/sonatype/sonatype-react-shared-components/pull/266/commits/3d9261cac84abbe4f371464bff300e6b3c071c9d

Reply:
> > Mostly I think that .sub-label should be consistently either inside or outside of labels and legends, I'm not sure that it matters which and I can switch all of the examples to inside.
> 
> Originally, the label text, sublabel, and input were all inside of the label. Then for screenreader support we changed it so the input and sublabel were outside of the label. Maybe we didn't have to move the sublabel at that time, I'm not sure. But as it stands that history means we are already supporting two different ways of doing them, I'd rather not add a third by moving the sublabel back inside of labels. If you want them consistent I'd rather move legend sublabels to the outside. But if that's what you do it needs to be changed in `NxFieldset` as well.

I moved them back outside here: https://github.com/sonatype/sonatype-react-shared-components/pull/266/commits/61f6808a091263c2deebf6897a7d716fc1a95e5d and modified `NxFieldset` as well, which was good because now it is consistent with `NxFormGroup`: https://github.com/sonatype/sonatype-react-shared-components/pull/266/commits/7235813987a69b379aaa876904baea86fbb192ff